### DISCUSSION TITLE
Мысли по поводу тотума.

### DISCUSSION THREAD

**Discussion by LostAlly** [totumonline/totum-ru-issues-and-discussions#513] [Неопределенное]
Мне нравится Тотум
Видно, что Алексей очень болеет за проект.
Но по моим наблюдениям:
1. Вокруг Тотума не развивается сообщество ни пользователей ни разработчиков, один более менее заметный и активный разработчик решений на Тотуме сторонний но он занят постоянно, не пробиться ) А из сделаных и опубликованных проектов самый внешне грамотный (фабрика мебели) у пользователя очень продвинутого и умеющего очень хорошо выстраивать процессы или очень приближенного к разработчикам тотума или целой команды.

2. При позиционировании Тотума как базы данных для непрограммистов все вокруг него сделано программистами для программистов начиная с этого «форума» заканчивая документацией и интерфейсом.
Продвижение по моим наблюдениям нацелено так же на  неявную когорту разработчиков, которая и не разработчики и не просто пользователи(но для такой категории тотум слишком сложен по сравнению даже с таблицами, и в сравнении с ними очень высокий грог входа и начального использования)

Может быть стоит максимально притормозить развитие функциональности платформы и сделать готовые «конфигурации» как в 1с с простой но законченной функциональностью - CRM / склад / финансы / производство / продажи, с возможностью комбинировать их в единое целое. Кому-то хватит базовой функциональности кто то захочет расширить/модифицировать функционал под свои задачи.
Тогда можно будет предлагать систему предпринимателям, а не айтишникам на Хабре. Тогда и облако будет интересно.
Перенести этот форум с gh на платформу для обычных людей а не айтишников. Фб/ВК/телега

На сколько я понимаю реальных проектов на много больше чем опубликовано, может как-то мотивировать людей писать о своих проектах?

Прошу не воспринимать данное сообщение как критику, я реально переживаю за проект. Понимаю финансовую составляющую, я прочитал все дискуссии здесь.

Огромное спасибо команде тотума за проделанную и проделываемую работу. Возможно мои слова натолкнут вас на новые мысли.

**Comment by ttm-alexeypolunin** [comment]
Итак, ща по очереди все разберем.

Хорошо, что вам нравится Тотум, как минимум вы и есть его целевая аудитория. Есть еще ~180-200 проектов в которые заходит Создатель. Мы это знаем тк Тотум отправляет раз в 2 недели запрос на получение нотификаций от нашего центрального сервера. Он ничего не передает, кроме версии системы и уникального ключа. Больше мы ничего про эти установки не знаем.

Я знаю есть часть, которая работает внутри закрытых сетей, без доступа в интернет. Про них ничего не знаем. Только когда нам приходят вопросы по этим установкам и иногда донаты. Иногда, весьма приличные. Самый большой разовый донат на текущий момент — 300 т.р.

Много это или мало — и так и так. Много, с учетом нашего отрицательного маркетинга. Мало с точки зрения бизнеса.

Почему мы не делаем маркетинг — потому что это было рано. Бессмысленно что-то масштабировать, если нет рабочей схемы, как это что-то будет зарабатывать.

+ Обучение — заказывают, но этого недостаточно.
+ Облако — иногда заказывают, но те, кто готов платить реальные деньги, хотят свой собственный сервер, поэтому они и пришли в Тотум.
+ Сервисы — иногда заказывают, но точно так-же, крупные потребители хотят их локальную версию (ее мы скоро начнем раздавать).
+ Готовые решения — это черная дыра для наших денег (чуть позже объясню почему это не работает).
+ Конечная разработка на заказ — это черная дыра для времени со средним результатом.

Что же будет сейчас — лицензии на сам Тотум, на его локальные установки. Это недорогие лицензии, но тут и будет понятно, кому это реально приносит пользу.

Дело в том, что я такого вида совет (разрабатывать конечные решения для обеспечения финансовой устойчивости проекта), получаю стабильно раз в неделю.

**Что на самом деле здесь происходит:** кто-то разработал себе решение, оно хорошо заходит и становится немного стремно, что Тотум возьмет и загнется (он же бесплатный). Я могу подтвердить, что если продолжит быть бесплатным — точно загнется. Но совет всегда предполагает, что заплатит кто-то другой :) не советующий (он то за что должен пострадать). 

_Без обид, тк возможно — это не ваш случай._

Но так не работает, все попытки взять денег на разработку проекта, предлагая какие-то сторонные фиговины — провальные. Дураков нет, фиговины всегда в таком случае выглядят как оверпрайс и без них можно обойтись. Платить должны за основной продукт, который получается выгоден в комплексе.

**Поэтому будет платный PRO.** Да, остается бесплатный MIT, но мы выковыряли оттуда все, что должно быть в PRO:

— файлы (все варианты)
— перенос частей решений (тк они с файлами связаны)
— -- смена стандартного лого тоже сама собой отвалилась :)
— формы
— анонимные таблицы
— продвинутые запросы к API
— карточки и канбан
— вид деревьями
— поворот таблицы

Да, в PRO будет лимит на 10 бесплатных пользователей, да, кто-то будет вдесятером работать от одного аккаунта (потом подвезем ограничение на одновременно количество сессий).

**Но глабально — это более-менее рабочий вариант.** Более, тк у нас уже есть лист записи на покупку лицензий после выхода 5. Менее, тк лицензия будет реально недорогая, ибо все можно взломать и с какой-то суммы, ломать, будет приоритетным вариантом.

**Немного про разработчиков и про разработку.**

Контрибьюторов нет, тк нам самим сложно их обслуживать (а профит сомнителен). Да и они не стоят в очереди тк Тотум для непрограммистов.

**Мало тех, кто хочет разрабатывать проекты на заказ.** Да, не просто так. Я про это выпущу большую статью на хабр в следующем месяце. Если коротко, то это адская работа требующая дофига скила и когда у человека есть такой скилл, он идет и работает в яндексе с десятикратно меньшим напрягом. Дело не в сложности кода, дело связанно с ролевым взаимодействием заказчик-подрядчик. Это и для дешевых проектов работает и для дорогих, просто у дорогих есть бабло это фиксить.

Поэтому есть вереница людей, которые берут проекты на разработку на тотум втихую — от своих знакомых, в той же области, что они работают, такого плана. Но они не хотят записываться публично как разработчики на Тотум — нафиг надо. И я их прекрасно понимаю.

**Когда нет этой пары — заказчик-подрядчик, все сразу проще и Тотум становится супер-эффективен. Самые большие результаты у тех, кто выделил полностью или частично человека изнутри своей компании на разработку базы.**

Что это за люди — обычно у них есть какой-то маленький бэк в делании чего-то в интернете. Это не _Маша-пятая-помошница-секретарши,_ для которой нужна система с одной большой красной кнопкой. Это люди, часто с большой инженерной или другой подготовкой. Как пример _лучшее описание сделанного проекта_ — там есть два, подходящих под описание. Один делал бухгалтер, другой инженер производства. Никто из них с нами не афилирован.

Кажется, что это не аудитория. Да, все так думают, именно в этом наше преимущество. Тк оказалось, что такой запрос есть. Когда я даю самую кривую рекламу на планете или пишу куда-нибудь статью, оттуда приходит много народу. Хабр в том числе читают не только программисты.

**Когда приходит бизнес — самое эффективное для него конвертировать одного из своих сотрудников,** уже понимающего, как там звездные пули в его конкретной компании летают. Это мы им и советуем, тк Тотум от 1С отличается тем, что если даже самостоятельно его сотрудник уделит месяц работы изучению — он сделает рабочий проект. А потратив 180 часов на 1с очно (именно такой у них базовый курс), ничего законченного сделать нельзя.

Это основной поинт и именно на тех, кто может выделять 60-150 К на систему в месяц мы и ориентируемся.

Как это написать, чтоб правильно считали. Нельзя же написать, для полу-программистов — это обидно (половинка от головы или от жопы). Напишешь для начинающих, так люди, которым это нужно, не воспринимают себя начинающими. Писать для тех, кто технический профессионал трали-вали – до конца не дочитывают. Также должно быть слово Программирование, тк те, на кого ориентируется Тотум ищут программирование.  Поэтому самое рабочее и понятно с полсекунды — для не программистов, но не просто так а в комплекте с База данных и форумом на GH, чтоб нафиг сразу отваливались те, кто хочет одну большую красную кнопку. Им не подойдет, они только нашего времени поедят, а его и так очень мало. Нет задачи собрать всех-всех-всех, как кольцо всевластия — это путь в могилу.

**Почему не работают конечные решения.**

Они работают в супер-стандартизированных областях. Типа пробитие товара на кассе, или запись к парикмахеру. Когда что-то сложное, то делая конечное решение надо выбрать узкую полоску возможностей, которые будут в этом конечнике. Это значит, что надо либо делать на очень узкую аудиторию, либо минимум 30% потребителю не подходит.

За счет чего конечное решение зарабатывает — за счет масштаба. А его нет, когда мы сделали на узкую аудиторию. Ок, давайте тогда на широкую, пусть ему хоть половина не подходит, главное чтоб дешево (что и компенсирует ту часть, которая не подходит).  Ок, а мы как разработчик тогда как заработаем. Работы у нас умножится — поддерживать его нужно, документацию написать, а продавать надо дешево.

Так что делать конечники — кратчайший способ похоронить проект.

Про комбинирование разных модулей я молчу — разберите бумер и мерс и попробуйте собрать одну тачку из них двоих. Вот это аналогично задаче комбинирования блоков. Проще с нуля написать что надо.

**Да, это не бесплатно и да у Тотума есть порог входа. И мы будем работать над его понижением.** Но не через конечники, а через ChatGPT, который будет супер-справочной системой и который будет писать код на Тотум.

**Про разработку функционала**

Сейчас будет некоторая пауза связанная со сторонними причинами, но останавливать разработку конечно нельзя. Необходимо всегда быстро бежать, что бы оставаться на месте. Есть много смешных случаев, когда я разговаривал с человеком, говорившим **хватит разрабатывать** и через неделю, просившим добавить N тк ему сложно обойтись без этого в его проекте, а добавить вроде несложно. Поэтому список того, что еще должно быть в Тотум — большой. Особенно большой он в части _доработка на продакшене работающего проекта дешево_.

**Про людей написавших про свои проекты публично**

Для этого мне нужно было сопровождать их проекты в телеграм от 3 до 6 месяцев и потом некоторых по несколько раз просить написать. Вы то сами написали? Разные причины почему не пишут. Основные:
— проекты внутри обычно кривые (не путать с неработающие), неудобно как-то такое выставлять
— да и это надо напрягаться, тереть на скриншотах данные
— да и разрешение у босса надо спрашивать (те, кто сотрудник), а боссу это зачем
— босс то как раз никому рассказывать не хочет, чтоб конкуренты не увидели твари (это мотивация реального человака кста)

**ФБ/ВК/Телега**

Фб надо заниматься, ровно на тот объем, который нужен, что бы давать там рекламу. Конечно это eng версия.

То же самое X. Здесь можно побольше писать, анонсов всяких, но это много усилий и мы заканчиваем сейчас последние проекты, которые мы брали для поддержания штанов, пока делались лицензии и будем делать X и больше учебных материалов.

ВК — ну гайс, нет.

Телега — я не делаю общую группу в телеге, тк это работа ее модерить. Также телеграм в части объяснения как должно работать — ограниченный вариант. Мы будет продолжаь делать это на GH тк потом можем конвертировать ответы на английский и засунуть в поисковую систему и ChatGPT.

**В качестве заключения**

Чуваки, мир стремительно меняется. Теперь айтишник — это не тот, кто профессионально айтишник. Теперь любой профессионал должен быть немного айтишником, а любая компания немного айтишной компанией. Теперь ко всем знаниям мира (почти) можно получить доступ за 5 секунд (в РФ со сложностями). Теперь пороги входа можно преодолевать в 1000 раз быстрее — я это на себе ощущаю, когда изучаю как например, nginx настраивается, или что-то там еще на сервере. Тотум в этом мире имеет некоторое преимущество в том, что код и структура не такие сложные как могли бы быть, и даже нагенерив код на тотум в GPT, можно будет чекнуть, че же он там делает, имея диплом по биологии, а не по компьютер саенс.

**Reply by ttm-alexeypolunin** [reply]
**Пока жую сосиску — расскажу еще умных мыслей.**

Есть еще категория желающий получить конечное решение — это бизнес, обычно небольшой, но с большим запросом на автоматизацию.

**Почему запрос большой** — потому что они там не могут определится с тем, что им реально нужно автоматизировать, не могут расставить приоритеты. Нужно то и се и двадцать-пятое. Параллельно с этим есть пожелание к небольшому чеку, что понятно, бизнес то небольшой. Эти ребята быстро понимают, что если они сами сядут это делать, то зашьются. Да и сложно разрабатывать самому, когда у тебя запутанные противоречивые требования. 

**Сразу идея — я пойду и закажу, есть же на планете профессионалы.** Предположим вот он есть профессионал, который должен иметь реально 10 летний опыт в должности исполнительного директора в 10 разных отраслях, чтоб вообще разобраться (пока даже без какого либо программирования) в 100500 непонятных пожеланий неизвестного бизнеса с неизвестными заморочками.

Сколько это стоит?  Прикинем по времени. Ощутимый проект на Тотум около 200 часов, из который 50 — это вот это выяснение, как же там должно работать.

50 — это треть доступного рабочего времени, причем это должно быть потрачено самое продуктивное время, тк обучение (а это обучение чужому бизнесу) требует супер-концентрации. Чел с такой квалификацией получает 300К только в путь. Это будет 100К затрат на всякие совещания, предварительные и в процессе, при нормальном раскладе. А в таких случаях тк там переавтоматизация, то это в 2 раза сложнее — это 200К (еще ничего не сделали).

**Для подрядчика — это нерабочий вариант.** Предположим даже если эти 200 платят, так их будут требовать обратно, когда после 3-х месяцев переговоров подрядчик скажет что-то типа «ну знаете, со всеми условиями это будет стоить 1.5М».

**Но обычно до этого не доходит, тк бюджет у таких потребителей что-то в районе 30 тр. на все.**

**Появляется следующая идея** — а давайте будет типа законченное решение. Оно будет стоить 15 тр. и оно решит 2 проблемы:

— проблему денег (оно же конечно и на 10000 покупателей распределяется)
— проблему того, что самим непонятно, что и как автоматизировать

**Но, там есть важно но.** Требование следующее, оно прослеживается в переговорах: это должно быть типовое конечно решение, но оно должно **точно подходить под мой конкретный бизнес.**

Что есть утопия — нет такого на планете земля, противоречит законам физики.

Тк сделана уже тьма конечных решений — 1С сделал 100500 разных. Если можно взять типовое — то оно у них есть, можно пойти и взять, или МойСклад есть или ТвойМойБизнес или как он там называется. Не надо рассказывать, что у них там что-то не подходит и сложно переделать — это и есть типовые решения с их фундаментальными недостатками. Те если сделать на Тотуме типовое CRM, то оно будет таким же черезжопным и неподходящим как и существующие. Типовой склад — он будет хуже чем МойСклад.

Преимущество Тотума именно в Кастоме, но это не всем подходит. **И чем меньше бюджет, тем меньшему количеству бизнесов это подходит.** 

**Супердилерство**

Есть особая форма приседания на уши. Подход заключается в следующем: приходят к разрабам чуваки и начинают рассказывать такую пластинку:

— на рынке Х все решения говно
— мы одни знаем как надо
— у нас есть экспертиза в маркетинге
— давайте пополам поделим ответственность
— с вам разработка, а с нас маркетинг и тайные знания
— мы вам за разработку конечно не платим
— мы же потом будем делать супер-маркетинг
— и все станем миллионерами

блаблабла... что означает в переводе на русский _«нам нужно кастомное ПО, но платить мы за него не хотим, давайте вы запилите нам его бесплатно, может мы потом что-то и будет пытаться продавать, но скорее всего продавать мы будем то, что вы пилите кастомный софт бесплатно, мы же партнеры»_ и там еще в комплекте приложены приглашения в ресторан попить кофе в середине недели и приятные беседы, в которых демонстрируется безмерная заинтересованность и прилагаются рассказы о невероятном успехе платформы, но обязательно, при следовании правильному курсу партии :)

**Вот с конечными решениями где-то близко. Вы запилите, мы конечно купим, но не дороже 15 тр. и только тогда, когда все будет работать так, как нам надо**

**Reply by LostAlly** [reply]
Огромное спасибо за такой развернутый ответ.
Общая картина более менее стала понятна.

Не публиковал информацию о готовом решении так как его нет, я нахожусь на стадии тяжелого освоения, на данный момент пока создаю справочники и обмены с внешними сервисами.  Медленно и упорно двигаюсь вред. Про смену политики лицензирования читал в постах ранее и считаю ее положительным движением, так как увеличивает стабильность проекта, и мелким пользователям вроде меня дает бесплатно и да сейчас ситуация, что платит кто то другой.

Про большой запрос это про меня, точнее будет сказать так - пришло время более серьезного учета чем Гугл таблицы низкого уровня сложности, но из-за разнообразия процессов формально описать их для разработчика требует довольно много времени, а я даже пока не могу записаться на бесплатную ни на платную консультацию, так как понимаю, что нельзя придти на нее неподготовленным, это будет пустая трата времени для обеих сторон.

По поводу мелких решений видимо я не полностью раскрыл мысль и это за меня отлично сделал gorbatov, очень мало примеров которые можно посмотреть повертеть, и блоки с базовым функционалом дадут ту платформу для начала изучения и которую или переделываешь с набором опыта или модифицируешь под себя, мастер классы конечно хорошо но поковыряться и покрутить законченное решение дорогого стоит. Для меня сейчас источником информации на первом месте стоит просмотр скринов готовых проектов потом лезу в справочные материалы. Но пока продвижение идет очень медленно…

Планируется ли второе видео про которое говориться в конце урока по древовидным таблицам? )))

**Reply by ttm-alexeypolunin** [reply]
Если слишком много вариантов бизнес-процессов, так много, что не получается их упаковать в схему с 70 квадратиками — надо унифицировать бизнес-процессы. Никакое программное обеспечение не разрулит больше. Не потому что компьютер не справится по мощности, а потому что и разработчик и пользователи люди. В нем будет столько нюансов, что разработчик запутается, а пользователи очень быстро забудут, как делаются вещи, которые они делают иногда.

Это и есть — переавтоматизация. Когда все самые мелкие ньюансы стремятся запихнуть в ПО. Это чаще всего возможно для каких-то простых процессов. Например кассир на кассе пробивает котлеты — там есть фикс цена, штрихкод, количество, может возврат, но не больше. Когда у нас менеджер в офисе и сделка с 500 параметрами — то не надо там везде делать жесткие блокировки — они не работают. Как-то более общо.

Также надо соизмерять количество нюансов с оборотами. Если обороты компании миллиард в месяц — то там есть запас на нюансы. Если 500К — то наверное надо подойти к вопросу попроще. А то может и вообще автоматизация не нужна. Работает же сейчас как-то. Если не работает, то что конкретно? Это и нужно автоматизировать, остальное потом. А может оно и не из-за автоматизации не работает, тогда и это автоматизировать не стоит.

Те когда мы видим вот такое 👇 то это 1.) почти предел сложности блока; 2.) даже по лайтовой ставке, даже на Тотуме, это стоит ~400К

Понятно, что здесь не видно справочников, обмена и прочей обвязки, всего там ~35 таблиц и ~500 полей. И времени ~160 часов.

**Это реальный проект и запускался он на производстве с 80 часа разработки! И после запуска эта схема изменилась на 35%, те то, как это представили на предварительном этапе — не работало в реальных условиях.**

![primer](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/55755565/c0c3edd3-a5c1-488d-804f-d85a65622935)

**Упрощайте, и не стоит начинать с синхронизаций, перенесите данные через .csv и начните работать.**

**Пока заказчик (не важно он сам у себя заказал или у кого-то) не приступил к эксплуатации решения в реальных условиях — он не может сформулировать реальные требования. Всегда становится понятно, как должно работать и что надо делать только, после начала эксплуатации.**

Ко мне приходили люди с проектами (которые им на php делали или pyton), где даже не в деньгах было дело. Их проекты выглядели так:

— их пилили 2-3 года
— они не эксплуатировались в реальных условиях
— затраты там обычно по 1М/год те около 3М всего на проект
— ни один из них не был готов к эксплуатации
— в каждом было дофига модулей
— но ни один не работал до конца, везде были разобранные куски

И самое интересное — **они на самом деле не хотели запускать их в эксплуатацию,** тк если бы они это сделали, то стало бы четко понятно, что они потратили кучу времени и денег на фигню. Эти люди выдумывали новые и новые модули, что бы всегда иметь возможность объяснить себе — почему надо отложить запуск.

Это приблизительно как если бы Тотум до сих пор не был открыт публично и мы бы что-то втихую пилили — это мертвяк.

**Есть такое эмпирическое правило Тотума: если вы не запускаете первый модель в реальную работу через 100 часов работы на проектом максимум — этот проект уже закончился провалом.**

**Comment by gorbarov** [comment]
Боль в каждом абзаце ответов (

Но какое же крутое решение вы запилили! Благодарность всей вашей команде. Пусть это и будет оффтопик.

А по сути две мысли:

**Наблюдение на нашем примере**
От себя могу сказать однозначно - если выделить человека, который освоит Тотум, то можно делать очень крутые вещи.
У нас сейчас в процессе автоматизация диджитал агентства (т.е. нас самих) с интеграцией с 1С и гуглодокументами. Получается круто. Была изначальная мысль, что мы будем это как готовый продукт asis поставлять... но это действительно утопия.

Если и будем, то только с внедрением, но уже сейчас понятно, что это будет довольно дорогое внедрение. Потому что у всех свои немного (иногда значительно) отличающие процессы, хотя в итоге всё и сводится к управленческому учёту, который плюс/минус у всех похожий и набору других понятных отчётов.

Вероятно, сделаем лендинг для конкурсов где будем хвастаться как можно довольно быстро (по сравнению с кастомной разработкой) сделать подобное решение. Но что-то подсказывает, что за деньги, которые действительно стоит такое решение кастомизировать - покупателей не найдётся.

У кого нету автоматизации вообще - им будет дорого, а кому не дорого - уже сделали сами, скорее всего. Это, безусловно, надо проверить будет. Но что-то подсказывает, что так и будет.

И это речь про плюс/минус понятное решение в нашем собственном бизнесе. Чё говорить про готовые решения для других бизнесов.

**Каталог шаблонов**
Но в целом я мысль топикстартера поддерживаю. Только речь не про готовые решения, а про набор неких шаблонов с которых можно стартовать. Например блок CRM, но который нужно самому допилить под свои нужды, P&L, движение денег, складской учёт и т.д.

Больше всего времени мы потратили и консультации брали ровно для того, чтобы разобраться "а как надо", через пару месяцев проб и ошибок стало понятно и процесс пошёл, но вот такой каталог решений бы значительно ускорил этот процесс.

Как пример https://n8n.io/workflows/. Похож на Тотум чем-то. Тоже сложный продукт, если ты такого рода сервисами не пользовался, тоже модель "можешь юзать бесплатно у себя на серваке" и схожие проблемы "а чё вообще с ним можно сделать", "как бы мне сделать вот так вот или вот так".

Отчасти это решается в видосах, где есть примеры как подключиться к битриксу, амке и тд. Но это совсем небольшие кирпичики, а нужны большие блоки с законченным функционалом. И достаточно простым способом накатить такой блок к себе в Тотум.

Увидел то, что тебе надо, накатил, посмотрел как сделано, доработал под себя. И сразу разобрался, освоил лучшие практики и идеологию системы понял.

Тогда стартовать с нуля будет сильно легче. Нам-то повезло, я видел проект, который делал Алексей и много времени потратил анализируя те или иные решения. А как это делать не имея перед глазами больших готовых блоков функционала - с трудом представляю.

**PS.** На истину в последней инстанции не претендую. Тоже переживаю за Тотум, продукт-то офигенный! Не выдержал, решил поделиться всё-таки мыслями.

**Reply by ttm-alexeypolunin** [reply]
Не то что бы боль, но накопилось очень много весьма конкретных знаний (практических). Понятно, что каждый кусочек этой практики был обеспечен реальными усилиями :)

Я еще не видел примера, когда кто-то делал проект для себя и был успешен в его продаже как есть на рынке. Это разные подходы. Что бы на рынке — надо сразу обобщать, что не стыкуется вообще с идеей проекта для себя. Он то как раз наоборот должен быть супер кастомным.

Меня как-то хотели нанять люди, которые сделали производственное ПО для конкретной фабрики и пошли его задешево продавать всем. На практике это вылилось в то, что они дотировали доработку под каждого следующего клиента, делая проект все более и более громоздким, что удорожало каждую следующую доработку. Те они теряли все больше денег, на каждом следующем клиенте. Спрашивали, как из этого выйти — никак.

Когда проект сложный — то доработка получается либо говеной (как это в 1С мы видим повсеместно), либо реально проще сделать с нуля (что и делают). На 1С там еще есть сомнения, получится ли с нуля дешевле (часто не получается), а на Тотуме получается — для того и сделан. 


**Учебные шаблоны**

Да, их надо сделать именно как **учебные шаблоны.** Брать их в работу в реальные проекты — врядли. Их будут смотреть как референсы. Это достаточно дорого, их разработка. И проблема даже не в деньгах — а в том, что шаблоны хотят те, кто не хочет платить. Получается, что мы вложим кучу усилий в ту часть аудитории, с которой нет ресурсного возврата в ближней перспективе.

Если бы их покупали например... но мы все прекрасно понимаем, что их не покупают. Я видел (забыл у кого) как люди начали делать магазин таких решений, для своей платформы. Видно как они сделали первое и второе с хорошими описаниями, рекламой, третье — уже хуже, чутвертое — совсем дохлое и пятое — незаконченный обгрызок. Просто к пятому им уже стало понятно, что это не покупают. Цены были типа 12-13 тр. за решение. **Не просто так я их не делаю.**

У кого они есть — у проектов с инвестициями в $1.5М. Там можно хоть двух человек посадить их клепать. Что они и делают. Как это выглядит — они сделали одно из 5 табличек и 150 раз его по разному назвали. **Классно работает для маркетинга** — мне стабильно люди шлют на них ссылку. Самое интересное, что никто из приславших ни одно из них не открывал, все смотрели только на список с названиями и такие «вау». Ну это вестерн ворлд, здесь действительно так принято — маркетинг ферст.

Поэтому мы ими будем заниматься, но потом, когда можно будет свободно человека выделить работать на менее мотивированную аудиторию и иметь возможность списать все эти расходы в никуда, не ожидая возврата.

**Пока мы будем заниматься обучением GPT** — это выглядит более интересным направлением приложения усилий.

**Reply by ttm-alexeypolunin** [reply]
n8n кстати, заимел кучу примеров, когда в 2021 поднял $12M Series A round led by Felicis Ventures

До этого было поскромнее, тк у них было всего $1.5M Seed round led by Sequoia and firstminute Capital, with the participation of Runa Capital, Tiny VC and System.One

37К звезд на GH. Мне звезда по рекламе стоит 10 баксов. Те 370К в рекламу. За 1.5М мы бы обогнали их по результатам раза в 3-4. Как-то так

**Reply by ttm-alexeypolunin** [reply]
Это была платная консультация и мне чел рассказывает, что вот у _«я забыл что за БД»_, документация лучше чем у нас. Примеры более понятные и их побольше — раза в 1.5 где-то.

Я говорю: ну ок, давай смотрим объем инвестиций... $300М.
И тут чел тоже признал: Что за 300М, могло быть реально получше.

**Reply by gorbarov** [reply]
Понятное дело, что с деньгами жить намного комфортнее чем без денег. Что в частной жизни, что в бизнесе. 

Сколько надо времени, чтобы объяснить что такое Тотум при отсутствии широкоизвестных конкурентов на рынке (учитывая, что рынок-то не такой уж широкий)? "Low-code конструктор ERP систем" - вот так можно назвать Тотум, при условии, что человек знает значение всех слов по-отдельности из этого предложения. 

Именно поэтому шаблонные решения нужны всем. В эту категорию попадают и те кто готов платить и те кто не готов платить и те кто готов советовать (но не понимает насколько крутое решение) и те кто ищут решение для себя и тоже не понимают мощь и гибкость решения, потому что с готовыми решениями можно только по скринам ознакомиться.

Возможно, мы выложим наше решение asis бесплатно, а платными будут только консультации по внедрению и изменению бизнес-процессов внутри компании.

Я знаю, что у вас стройная картина мира и вы понимаете что надо делать, а что нет для своего продукта.

Но у меня такое же стойкое мнение, что шаблоны существенно бы расширили воронку, где на дне оказываются те, кто сделал/заказал на Тотуме решение для своего бизнеса. А воронка штука такая, на 10 халявшиков приходит 1 с деньгами, в этом в общем-то и есть цель бизнеса. Отсеивать зёрна от плевел, лишь бы было что-то на входе.

Без учебных/реальных/любых на самом деле примеров, которые можно поставить себе и потыкать, где есть целиком несколько бизнес-процессов порог входа существенно выше, чем с ними. А на входе риск потерять гораздо выше. Особенно для сложных решений.

Ну и очень интересно посмотреть какие результаты будут с GPT. Сможет ли он писать код за человека )))

**Reply by gorbarov** [reply]
> Мне звезда по рекламе стоит 10 баксов. 

Закинул в чат разработчиков, накинули 10 звёздочек в GH. Мелочь, а 100 баксов сэкономили )))

**Reply by ttm-alexeypolunin** [reply]
Спасибо!
